pkgs/top-level/aliases.nix

```diff
@@ -158,7 +158,7 @@ mapAliases ({
   firefoxWrapper = firefox;           # 2015-09
 
   firestr = throw "firestr has been removed."; # added 2019-12-08
-  fish-foreign-env = fishPlugins.foreign-env; # added 2020-12-29
+  fish-foreign-env = throw "fish-foreign-env has been replaced with fishPlugins.foreign-env"; # added 2020-12-29
```

Comment:
I'm unclear on the `aliases.nix` conventions -- should the date here be updated (since the alias contents have changed)?

Reply:
I've seen a few "added …, modified …". I added today's date so that there's no ambiguity.